handouts/Session_14.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Phylogenies in Computational Historical Linguistics\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Introduction\n",
+    "\n",
+    "### 1.1 Inner and Outer Language History\n",
+    "The origins of phylogenetics go back to the 19th century and its principles haven't changed much since. Among the lesser known early linguists there is Georg von der Gabelentz (1840-1893). The following description of phylogenetic reconstruction which is based on his words from 1891 may still hold true:\n",
```

Comment:
I am inclined to not use "haven't" and similar things in texts. Rather use "have not". 
Furthermore, "lesser known" -> "less well-known"